packages/expo-notifications/android/src/main/java/expo/modules/notifications/managers/SchedulersManagerProxy.java

```diff
@@ -0,0 +1,63 @@
+package expo.modules.notifications.managers;
+
+import android.content.Context;
+
+import org.unimodules.core.interfaces.Function;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+import expo.modules.notifications.schedulers.Scheduler;
+
+public class SchedulersManagerProxy implements SchedulersManager {
```

Comment:
All the managers are related to schedulers, right? It would be more obvious if they were put inside `schedulers` package. 🙂